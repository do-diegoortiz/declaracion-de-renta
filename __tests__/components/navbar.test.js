import React from 'react'
import { mount } from 'enzyme'
import { create } from 'react-test-renderer'

import Navbar from '../../components/navbar/navbar'

describe('<Navbar />', () => {
  
  test('It render without crashing', () => {
    const navbar = mount(<Navbar />)
    expect(navbar.length).toEqual(1)
  })

  test('Check the component UI', () => {
    const navbar = create(<Navbar />)
    expect(navbar.toJSON()).toMatchSnapshot()
  })
})
