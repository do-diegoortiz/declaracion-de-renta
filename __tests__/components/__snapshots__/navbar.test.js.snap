// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Navbar /> Check the component UI 1`] = `
<div>
  <img
    alt="Logo"
    src={Object {}}
  />
  <svg
    viewBox="0 0 500 70"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M0.00,49.98 C162.52,104.90 266.37,-13.53 500.00,49.98 L500.00,150.00 L0.00,150.00 Z"
      fill="#FFF"
      fillOpacity="1"
    />
  </svg>
</div>
`;
